From 2e30874fe515047b8373e4462a0aa1f113ee0b0a Mon Sep 17 00:00:00 2001
From: Paul 2 <pag509@york.ac.uk>
Date: Thu, 13 Feb 2014 12:35:40 +0000
Subject: [PATCH] Commented print statements

---
 .../components/FlightPlanComponent.java            |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ini-atc/src/seprini/controllers/components/FlightPlanComponent.java b/ini-atc/src/seprini/controllers/components/FlightPlanComponent.java
index 00ee1aa..7a850c0 100644
--- a/ini-atc/src/seprini/controllers/components/FlightPlanComponent.java
+++ b/ini-atc/src/seprini/controllers/components/FlightPlanComponent.java
@@ -29,10 +29,10 @@ public class FlightPlanComponent {
 		// Initialisation of parameters required by flightPlanWaypointGenerator.
 		ArrayList<Waypoint> flightPlan = new ArrayList<Waypoint>();
 		Waypoint entryWaypoint = setStartpoint();
-		System.out.println("Entry point " + entryWaypoint.toString());
+		//System.out.println("Entry point " + entryWaypoint.toString());
 		Waypoint lastWaypoint = setEndpoint(entryWaypoint, 620 );
-		System.out.print("Exit point " + lastWaypoint.toString());
-		System.out.print("\n \n");
+		//System.out.print("Exit point " + lastWaypoint.toString());
+		//System.out.print("\n \n");
 		
 		// entryWaypoint immediately added to aircrafts flightPlan.
 		flightPlan.add(entryWaypoint);
-- 
1.7.9.5

